experimental/handoff-library/csharp_dotnetcore/samples/LPProxyBot/LPCredentialsProvider.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LivePersonConnector;
+using Microsoft.Extensions.Configuration;
+
+namespace LPProxyBot
+{
+    public class LPCredentialsProvider : ICredentialsProvider
+    {
+        public LPCredentialsProvider(IConfiguration configuration)
+        {
+            LpAccount = configuration["LivePersonAccount"];
+#if RELEASE
```

Comment:
Do we need conditional compilation here? People can use the appsettings.json transformation perhaps?

Reply:
Removing